assembly.html.markdown

```diff
@@ -0,0 +1,218 @@
+---
+language: assembly
+filename: learn.asm
+contributors:
+    - ["", ""]
```

Comment:
To clarify, this is where your name (real or whatever) and website should go, e.g. `["Th3G33k", "https://github.com/Th3G33k"]`